### PR TITLE
OSDOCS-7799: Migrated a topic about using Azure AD for group claims

### PR DESCRIPTION
**Pull request by michaelryanmcneill**
Version(s):
4.13+

Issue:
[OSDOCS-7799](https://issues.redhat.com/browse/OSDOCS-7799)
[OSDOCS-7800](https://issues.redhat.com/browse/OSDOCS-7800)

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:
This consolidates the documentation migration for [OSDOCS-7799](https://issues.redhat.com//browse/OSDOCS-7799) and [OSDOCS-7800](https://issues.redhat.com//browse/OSDOCS-7800)
Supersedes #64966 and #64888  

Fixes #64966 Fixes #64888

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+//Brought into ROSA product docs 2023-09-18
+// ---
+// date: '2022-09-19'
+// title: Configure ROSA to use Entra ID Group Claims
+// authors:
+//   - Michael McNeill
+//   - Paul Czarkowski
+// ---
+
+This guide demonstrates how to configure Microsoft Entra ID (formerly Azure Active Directory) as the cluster identity provider in {product-title} (ROSA). This guide will walk through the creation of an Microsoft Entra ID (Entra ID) application and configure Red Hat OpenShift Service on AWS (ROSA) to authenticate using Azure AD.
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc:19]:
```suggestion
Use this tutorial to configure Microsoft Entra ID (formerly Azure Active Directory) as the cluster identity provider in {product-title} (ROSA). This tutorial will walk through creating a Microsoft Entra ID (Entra ID) application and configuring ROSA to authenticate using Azure AD. 
```

File: cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+// date: '2022-09-19'
+// title: Configure ROSA to use Entra ID Group Claims
+// authors:
+//   - Michael McNeill
+//   - Paul Czarkowski
+// ---
+
+This guide demonstrates how to configure Microsoft Entra ID (formerly Azure Active Directory) as the cluster identity provider in {product-title} (ROSA). This guide will walk through the creation of an Microsoft Entra ID (Entra ID) application and configure Red Hat OpenShift Service on AWS (ROSA) to authenticate using Azure AD. 
+
+This guide will walk through the following steps:
```
Comment (jneczypor) [inline cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc:21]:
```suggestion
This tutorial walks you through the following steps:
```

**Review by jneczypor — COMMENTED** [review]
Looks good for peer review! I did suggest changing "guide" to "tutorial" in a couple of places to match our new section.

**Comment by jneczypor** [comment]
Peer review done! Looks good @michaelryanmcneill!

**Comment by michaelryanmcneill** [comment]
Thanks @jneczypor! @EricPonvelle once the build completes, this should be safe to merge!

**Pull request merged**